tech-specs/multi-proof-and-verifiable-compute/how-to-build-with-multi-proof.md

```diff
@@ -1,12 +1,12 @@
 # How to Build with Multi-Proof
 
-With Multi-Proof JS SDK, things would become pretty easy to run a task and generate TEE-proof for it. You still can do it with our [Rust SDK](https://docs.phala.network/references/ai-agent-contract-sdks/getting-started) with more flexibility.
+We have JS Runtime custom-built based on [QuickJS](https://bellard.org/quickjs/). JavaScript code can be deployed to Phala TEE workers. We have engineered a WASM virtual machine called [Wapo](https://github.com/Phala-Network/wapo) based on [wasmtime](https://wasmtime.dev/), to facilitate its execution in SGX with the Gramine SDK.
```

Comment:
Will we need to update this to WapoJS instead of QuickJS with the upgrade to runtime?

Reply:
We can keep QuickJS here before WapoJS is officially released